Sources/OperatingSystem.swift

```diff
@@ -26,4 +26,213 @@
     @_exported import Glibc
 #else
     @_exported import Darwin.C
-#endif
\ No newline at end of file
+#endif
+
+import C7
+
+public enum SystemError: ErrorProtocol {
+    
```

Comment:
But we have the module namespace. It will always be possible to disambiguate. Bu